docs/vmanomaly.md

```diff
@@ -0,0 +1,126 @@
+---
+sort: 20
```

Comment:
It is already taken by other page - see https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/url-examples.md?plain=1#L2